caldp/20220214/CALDP_20220214_COS_macos_py38_final.yml

```diff
@@ -0,0 +1,105 @@
+# delivery_name: CALDP_20220214_COS_final
+# creation_time: Mon Feb 14 15:00:00 2022
+# conda_version: 4.10.3
+# condabuild_version: 3.18.11
+channels:
+  - https://ssb.stsci.edu/astroconda-staging
+  - https://ssb.stsci.edu/astroconda
+  - http://ssb.stsci.edu/astroconda
```

Comment:
Defaults are used here, but conda-forge used elsewhere

Reply:
Added the `conda-forge` channel to just before the `defaults`.